startup.d/30-install-requirements.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 /root/.virtualenvs/chaos/bin/pip install -Ur requirements.txt
-apt-get -y install puppet
\ No newline at end of file
+apt-get -y install puppet default-jdk
```

Comment:
this should probably be a puppet thing so that we can manage which things get installed on the VPS instead of doing it from scripts.

the current line is there to bootstrap puppet. Similar to the current PR that messes w/ the requirements.txt to bootstrap ansible.